Dockerfile.ci

```diff
@@ -1114,23 +1114,28 @@ function check_airflow_python_client_installation() {
     python "${IN_CONTAINER_DIR}/install_airflow_python_client.py"
 }
 
+function initialize_db() {
```

Comment:
Extracted the DB init functionality out of the start API server function (tt was kind of an overload to have it there in the first place) and we may want to init even if we're running on an older version of Airflow which does not have the API server (e.g. 2.10)